server/api.js

```diff
@@ -77,10 +77,59 @@ async function getSensor(req, res) {
   }
 }
 
+const validateMessageClients = Validator.body(['braveKey', 'data']).exists()
+
+async function messageClients(req, res) {
+  const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
+
+  try {
+    if (validationErrors.isEmpty()) {
+      let pgClient
+      pgClient = await db.beginTransaction()
+      const clients = await db.getClients(pgClient)
+      await db.commitTransaction(pgClient)
+      const data = req.body.data
```

Comment:
When I read `data`, I was expecting something relatively complicated like JSON to be coming through here. Can you think of a more descriptive name?

Reply:
I could call it `message`!